src/content/docs/fr/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Base renforcée pour vos applications web.
+description: Les outils de création d'applications multiplatformes
+i18nReady: true
+template: splash
+hero:
+  image:
+    file: ../../../assets/logo-outline.svg
+  actions:
+    - text: Commencer
+      link: fr/2/guide/
+      icon: right-arrow
+      variant: primary
+    - text: Directives de Contrubution

```

Comment:
```suggestion
    - text: Règles de contribution

```
Can be also `Guide de contribution`. I think `Directives` is a too strong word here